.github/workflows/package-command.yaml

```diff
@@ -2,136 +2,102 @@ name: Package Command
-        run: |
-          $instrumentationKey = "${{ env.APPINSIGHTS }}"
-          $defaultPackageVersion = "${{ vars.defaultPackageVersion }}"
-          $solutionSupportedBy = "${{ env.SOLUTION_SUPPORTED_BY }}"
-          $solutionOfferId = "${{ env.SOLUTION_OFFER_ID }}"
-          $runId = "${{ env.RUNID }}"
-          $solutionName = "${{ needs.solutionNameDetails.outputs.solutionName }}"
-          $pullRequestNumber = "${{ github.event.client_payload.pull_request.number && github.event.client_payload.pull_request.number || github.event.client_payload.pullRequestNumber }}"
-          $isNewSolution = "${{ env.IS_NEW_SOLUTION }}"
-
-          Write-Host "RunId = $runId, defaultPackageVersion = $defaultPackageVersion, solutionOfferId = $solutionOfferId, SOLUTION_NAME = $solutionName, PULL_REQUEST_NUMBER = $pullRequestNumber, isNewSolution = $isNewSolution"
-          . ./Tools/Create-Azure-Sentinel-Solution/common/LogAppInsights.ps1
-          try
-          {
-            $customProperties = @{ 'RunId'="$runId"; 'PullRequestNumber'="$pullRequestNumber"; 'EventName'="Create Package"; 'IsNewSolution'='$isNewSolution';'SolutionOfferId'='$solutionOfferId';'SolutionSupportedBy'='$solutionSupportedBy';'SolutionName'='$solutionName'; }
-            Send-AppInsightsEventTelemetry -InstrumentationKey $instrumentationKey -EventName "Create Package" -CustomProperties $customProperties
-
-            if ($solutionName -eq '' -or $null -eq $solutionName)
-            {
-              Write-Output "::warning:: Package-Creation : Skipping as Solution name is blank."
-              Send-AppInsightsTraceTelemetry -InstrumentationKey $instrumentationKey -Message "Execution for create-package started and received blank solutionName for Job Run Id : $runId" -Severity Information -CustomProperties $customProperties
-            }
-            else
-            {
-              Send-AppInsightsTraceTelemetry -InstrumentationKey $instrumentationKey -Message "Execution for create-package started for Solution Name : $solutionName, Job Run Id : $runId" -Severity Information -CustomProperties $customProperties
+          token: ${{ env.GeneratedToken}}
 
-              $baseFolderPath = "/home/runner/work/Azure-Sentinel/Azure-Sentinel/"
-              Set-PSRepository PSGallery -InstallationPolicy Trusted
-              Install-Module powershell-yaml
-              ./.script/package-automation/package-generator.ps1 $solutionName $pullRequestNumber $runId $instrumentationKey $defaultPackageVersion $solutionOfferId $baseFolderPath $isNewSolution
-            }
-          }
-          catch
+      - id: checkAutomatedPR
+        name: check-Automated-PR
+        env:
+          BranchName: ${{ github.event.pull_request.head.ref || github.event.client_payload.pull_request.head.ref || github.event.client_payload.pullRequestBranchName }}
+        shell: pwsh
+        run: |
+          $pullRequestBranchName = "$env:BranchName"
```

Comment:
will this give you the branch name or it must be something like this ${{ env.BranchName}}. plz check